packages/common/src/ArcGISContext.ts

```diff
@@ -91,6 +92,10 @@ export interface IArcGISContext {
    * For ArcGIS Enterprise this will return `undefined`
    */
   hubUrl: string;
+  /**
+   * Url for the Hub OGC API
+   */
+  OGCApiUrl: string;
```

Comment:
For consistency, I think we should case this as `ogcApiUrl`. This is the casing that I've used in hub.js so far and it's also the casing we use in the actual API.